src/main/resources/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger/help-stashHost.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    The base URL of the Stash server REST API.
+    This is <strong>not</strong> a URL for git.
```

Comment:
I think this might still be ambiguous. An example would be useful, also I'd phrase this like `Note: this is not the git clone URL`.
Please see the description in readme file.

Reply:
Adding more emphasis.